.travis.yml

```diff
@@ -4,3 +4,11 @@ services:
   - docker
 script:
   - bash tools/travis.sh
+matrix:
+  include:
+    - env: OPAMWITHTEST=false
+    - env: OPAMWITHTEST=true
+    - env: OPAMWITHTEST=true RECURSIVE=--recursive
+  fast_finish: true
```

Comment:
I would move this line under `matrix:` to emphasize that this is valid for all entries in the matrix.